<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="SONG1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>SONG1</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Downloads/Eighth-Note-Double.webp</normaloff>../Downloads/Eighth-Note-Double.webp</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="SONG2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>SONG2</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Downloads/Eighth-Note-Double.webp</normaloff>../Downloads/Eighth-Note-Double.webp</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QSlider" name="MIX_RATIO">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
gridline-color: rgb(255, 0, 255);
selection-color: rgb(255, 0, 255);
selection-background-color: rgb(170, 0, 127);
border-color: rgb(255, 85, 127);
color: rgb(255, 85, 127);</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="TABLE">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>210</y>
      <width>271</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="RATIO">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>130</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Waree&quot;;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_window"/>
   </widget>
   <widget class="QMenu" name="menubrowse">
    <property name="title">
     <string>browse</string>
    </property>
    <addaction name="actionLoad_song"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menubrowse"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_window">
   <property name="text">
    <string>New window</string>
   </property>
  </action>
  <action name="actionLoad_song">
   <property name="text">
    <string>Load song</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>MIX_RATIO</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RATIO</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>434</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
